data/tech/aria/aria-rowcount_attribute.json

```diff
@@ -9,6 +9,46 @@
       "url": "https://www.w3.org/TR/wai-aria-1.1/#aria-rowcount"
     }
   ],
-  "assertions": [],
-  "date_updated": "2019-01-06"
+  "assertions": [
+    {
+      "id": "convey_row_count_all_rows",
+      "title": "convey the total row count when all rows are visible",
+      "rationale": "The user needs to know how many rows are in the whole grid / table",
+      "strength": {
+        "sr": "MUST",
+        "vc": "NA"
+      },
+      "css_target": "*[aria-rowcount]",
+      "operation_modes": [
+        "sr/reading"
+      ]
+    },
+    {
+      "id": "convey_row_count_hidden_rows",
```

Comment:
How do you feel about reducing `convey_row_count_all_rows` and `convey_row_count_hidden_rows` to one expectation? The results are the same, and the ARIA spec states "If all of the rows are present in the DOM, it is not necessary to set this attribute as the user agent can automatically calculate the total number of rows." That makes me think that `convey_row_count_all_rows` might not be necessary for us to test.

Reply:
Removed the all_rows expectation and test results. ✔️